examples/cpu/surface.cpp

```diff
@@ -17,20 +17,20 @@
 
 using namespace std;
 
-static const float XMIN = -8.0f;
-static const float XMAX = 8.f;
-static const float YMIN = -8.0f;
-static const float YMAX = 8.f;
+static const float XMIN = -32.0f;
+static const float XMAX = 32.f;
+static const float YMIN = -32.0f;
+static const float YMAX = 32.f;
```

Comment:
```
static const float XMIN = -32.0f;
static const float XMAX =  32.0f;
static const float YMIN = -32.0f;
static const float YMAX =  32.0f;
```

Consistency of space and 0 after decimal point. I added 0s cuz 2 of them had it.